gui/wxpython/startup/guiutils.py

```diff
@@ -279,6 +295,21 @@ def rename_location_interactively(guiparent, grassdb, location):
     return newlocation
 
 
+def download_location_interactively(guiparent, grassdb):
+    """
+    Download location online
+    """
+    from startup.locdownload import LocationDownloadDialog
+
+    loc_download = LocationDownloadDialog(parent=guiparent,
+                                          database=grassdb)
+    loc_download.ShowModal()
+    location = loc_download.GetLocation()
+
+    return location
+    loc_download.Destroy()
```

Comment:
This known by a poetic name, _unreachable code after return statement_. :smile: Flake8 would probably warn you about this, but in any case, flipping the lines should be enough so that the `Destroy()` is called before the function ends.

Reply:
Those lines were flipped.